backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/functions/PP_Product_BOM_Recursive.sql

```diff
@@ -61,7 +61,7 @@ $BODY$
 						where bom.M_Product_ID=bomLineProduct.M_Product_ID
 						and bom.IsActive='Y'
 						and bom.Value=bomLineProduct.Value
-						order by bom.PP_Product_BOM_ID
+                        ORDER BY bom.validfrom DESC
```

Comment:
Please, also order by the old PP_Product_BOM_ID (or other unique thing)  after ordering by validFrom so we cover the cases where 2 BOMs are valid from the same date.

Reply:
Modification done